apps/prairielearn/src/batched-migrations/20250214035153_course_instance_usages__submissions__backfill.sql

```diff
@@ -0,0 +1,26 @@
+-- BLOCK delete_old_usages
+DELETE FROM course_instance_usages
+WHERE
+  type = 'Submission'
+  AND date < $CUTOFF_DATE;
+
+-- BLOCK select_bounds
+SELECT
+  min(id),
```

Comment:
Nit: I believe this can safely be hardcoded to `1`, not that it really matters.

Reply:
Changed in eb33f488082c3022fb7acc6bb46b6851b52d4504